packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -95,13 +93,13 @@ describe('--schema from parent directory', () => {
     ])
     await expect(result).resolves.toMatchInlineSnapshot(`
 
-            ✔ Generated Prisma Client (0.0.0) to ./subdirectory/@prisma/client in 147ms
-            You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
-            \`\`\`
-            import { PrismaClient } from './subdirectory/@prisma/client'
-            const prisma = new PrismaClient()
-            \`\`\`
-          `)
+✔ Generated Prisma Client (0.0.0) to ./subdirectory/@prisma/client in XXms
```

Comment:
I'm not sure why the the indentation changed

Reply:
One mysterious thing for me with Jest ...